UPGRADE-5.0.md

```diff
@@ -82,23 +118,6 @@ EventDispatcher
  * The signature of the `EventDispatcherInterface::dispatch()` method has been updated to `dispatch($event, string $eventName = null)`
  * The `Event` class has been removed, use `Symfony\Contracts\EventDispatcher\Event` instead
 
-DependencyInjection
```

Comment:
if this PR is pending till 4.4; this should be applied in master before 4.3 is branched

Reply:
Here you go: https://github.com/symfony/symfony/pull/31372 I will revert this change as soon as this PR is merged into master.